extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js

```diff
@@ -128,7 +128,6 @@ fairDataPointPostDialog.prototype._editBaseUri = function(src) {
 		}
 		MenuSystem.dismissAll();
 		self._replaceBaseUri(newBaseUri, false);
-		getFairCatalogs(self._baseUriSpan.text(), self);
```

Comment:
Was this line supposed to be removed? Seems like the retrieval of the catalogs is not triggered from the apply event now.